python/tests/io/test_pyarrow.py

```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+# pylint: disable=W0212,W0613
```

Comment:
Nit: curious on how pylint update its code to warning mapping, from understanding perspective, it's easier to read `disable=protected-access`

Reply:
Thanks, I like that a lot. Just updated the code.